tests/compile/ctypedef_bint.pyx

```diff
@@ -0,0 +1,10 @@
+# distutils: extra_compile_args=-Werror
+# mode: compile
```

Comment:
Could you please move this file into `tests/run/` and turn it into a runnable test, using a function with (Python) doctest? It's nice that it compiles, but I would also like to be sure that it produces the correct result. :)

Please also add some additional tests that mix `bint` with a regular C `int`, return the result from a function (which should convert it to a Python bool), etc.